configuration/external_parameters.rst

```diff
@@ -40,13 +40,41 @@ the following ``VirtualHost`` configuration:
         </Directory>
     </VirtualHost>
 
+For Nginx web servers, the environment variables can be set with `fastcgi_param`_
+directive. For example in the configuration file where the ``fastcgi_params``
```

Comment:
By the way, is the part about the included `fastcgi_params` file mandatory? I mean is it important that this file is imported?